coordinator/src/main/java/org/apache/uniffle/coordinator/SimpleClusterManager.java

```diff
@@ -226,8 +226,11 @@ private void parseExcludeNodesFile(DataInputStream fsDataInputStream) throws IOE
   public void add(ServerNode node) {
     if (!servers.containsKey(node.getId())) {
       LOG.info("Newly registering node: {}", node.getId());
+      servers.put(node.getId(), node);
+    } else {
+      servers.get(node.getId()).update(node);
     }
```

Comment:
```suggestion
    ServerNode previousNode = servers.get(node.getId());
    if (previousNode == null) {
      LOG.info("Newly registering node: {}", node.getId());
    } else {
      node.setRegistrationTime(previousNode.getRegistrationTime());
    }
    servers.put(node.getId(), node);
```

Reply:
@xianjingfeng If we modified like this, the other property of ServerNode would not be updated, something like usedMemory, preAllocatedMemory, availableMemory, eventNumInFlush and so on.